Documentation/contributing/development/contributing_guide.rst

```diff
@@ -183,13 +213,63 @@ Getting a pull request merged
 
 #. Owners of the repository will automatically adjust the labels on the pull
    request to track its state and progress towards merging.
+
 #. Once the PR has been reviewed and the CI tests have passed, the PR will be
    merged by one of the repository owners. In case this does not happen, ping
    us on Slack.
 
-
-Pull request review process
----------------------------
+#. If reviewers have requested for changes and once those changes are done,
+   re-request a review for the reviewers that have requested changes. Otherwise,
+   those reviewers will not be notified and your PR will not receive any
+   reviews.
+
+Pull requests review process for committers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Every committer in the `committers team <https://github.com/orgs/cilium/teams/committers/members>`_
+   belongs to `one or more other teams in the Cilium organization <https://github.com/orgs/cilium/teams/team/teams>`_
+   if you would like to be added or removed from any team, please contact any
+   of the `maintainers <https://github.com/orgs/cilium/teams/maintainers/members>`_.
+
+#. Once a PR is open, GitHub will automatically pick which `teams <https://github.com/orgs/cilium/teams/team/teams>`_
+   should review the PR using the ``CODEOWNERS`` file. Each committer can see
+   PRs they need to review by filtering PRs they has been requested reviews for.
+   A good filter is provided in this `link <https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+review-requested%3A%40me+sort%3Aupdated-asc>`_
+   so make sure to bookmark it.
+
+#. Belonging to a team does not mean that a committer should know every single
+   line of code the team is maintaining. For this reason it is recommended
+   that, once a review has been given and if the committer feels another pair
+   of eyes is needed, the committer should re-request a review from the
+   appropriate team. In the example bellow, the committer belonging to the CI
```

Comment:
nit, voice:

```suggestion
   that, once you have reviewed a PR, if you feel that another pair
   of eyes is needed, you should re-request a review from the
   appropriate team. In the example bellow, the committer belonging to the CI
```

This makes it more direct, it's *your* responsibility, not a thing that happens outside of your control.